draft-regext-simple-contact.md

```diff
@@ -332,14 +254,130 @@ of strings. Each string MUST be conformant to the geo URI scheme as defined in [
 
 "geo" values SHOULD NOT be used with contact data when "kind" is "individual".
 
+## An Entity Example {#example}
+
+The following is an example of an RDAP entity using SimpleContact:
+
+    {
+      "objectClassName" : "entity",
+      "handle":"XXXX",
+      "sc_data": {
+        "kind" : "individual",
+        "individualNames" : [
+          {
+            "name" : "Dr. Joe User Jr., Ph.d.",
+            "lang" : "en-US"
+          }
+        ],
+        "roleNames" : [
+          {
+            "name" : "Registration Services Help Desk",
+            "lang" : "en-US"
+          }
+        ],
+        "organizationNames" : [
+          {
+            "name" : "ACME Pty",
+            "lang" : "en-US"
+          }
+        ],
+        "postalAddresses" : [
+          {
+            "completeAddress" : [
+              "Suite 300",
+              "123 Random Tree Name Street",
+              "Kalamazoo, MI 90125 US"
+            ]
+            "regionCode" : "MI",
+            "countryCode" : "US",
+            "postalCode" : "90215",
+            "lang" : "en-US"
+          }
+        ],
+        "emails" : [
+          {
+            "email" : "joe@example.net"
+          }
+        ],
+        "voicePhones" : [
+          {
+            "phone" : "+447040202"
+          }
+        ],
+        "faxPhones" : [
+          {
+            "phone" : "tel:+1-201-555-9999;ext=123",
```

Comment:
Just curious that like we have domain names, IP addresses, and ASNs for documentation purposes, are they "tel" numbers reserved for examples? (Sorry, was lazy enough not to check if these already are example "tel" numbers. :))

Reply:
That's good enough.